File: Year-2023/LeetCode-April-Challenge/BulbSwitcher.java
class Solution {
    public int bulbSwitch(int n) {
        return (int)Math.sqrt(n);
    }
}
public class BulbSwitcher{
    public static void main(String[] args) {
        int n = 1;
        System.out.println(new Solution().bulbSwitch(n));
    }
}

File: Year-2023/LeetCode-April-Challenge/BulbSwitcher.cpp
#include<bits/stdc++.h>
using namespace std;
class Solution {
public:
    int bulbSwitch(int n) {
        return sqrt(n);
    }
};

int main(){
    int n = 1;
    Solution *s = new Solution();
    cout<<s->bulbSwitch(n)<<endl;
    return 0;
}